src/compiler/es-next/hybrid-function.js

```diff
@@ -125,14 +127,28 @@ export default function compileHybridFunction (fnCode, dependencies = {}, callsi
     // NOTE: check compiled code for regenerator injection: we have either generator
     // recompiled in Node.js 4+ for client or async function declared in function code.
     if (REGENERATOR_FOOTPRINTS_RE.test(fnCode))
-        throw new APIError(callsiteNames.instantiation, MESSAGE.regeneratorInClientCode);
+        throw createRegeneratorInClientCodeError();
 
     if (!TRAILING_SEMICOLON_RE.test(fnCode))
         fnCode += ';';
 
-    var dependenciesCode = getDependenciesCode(dependencies, callsiteNames);
-
     var { polyfills, modifiedFnCode } = addBabelArtifactsPolyfills(fnCode);
 
     return `(function(){${dependenciesCode}${polyfills} return ${modifiedFnCode}})();`;
 }
+
+export function compileFunctionArgumentOfHybridFunction (argumentFnCode) {
+    // NOTE: it safe to use test run error here, because this code
+    // will be hit only if context test run validation already passed.
```

Comment:
```
    // NOTE: it is safe to use a test run error here, because this code
    // will be hit only if the context test run has already been validated.
```